Source/Python/MultiFabRegister.cpp

```diff
@@ -56,6 +57,7 @@ void init_MultiFabRegister (py::module & m)
                  bool,
                  bool
              >(&MultiFabRegister::alloc_init<std::string>),
+             py::return_value_policy::reference_internal,
              py::arg("name"),
              py::arg("dir"),
              py::arg("level"),
```

Comment:
We should probably also add `return_value_policy::reference_internal` for the other methods that return a `MultiFab`, including e.g. `alias_init`